error: 'move' is not a member of 'stdlib'

When building a arduboy project with platform io it throws a dozen of errors with:
```
src/Deque.h: In member function 'bool Deque<Type, Capacity>::prepend(const ValueType&)':
src/Deque.h:265:21: error: 'move' is not a member of 'stdlib'
```

Sadly I don't know nothing about this to help.

--- a/Deque.h
+++ b/Deque.h
@@ -19,6 +19,7 @@
 #include <stdint.h>
 
 #include "TypeTraits.h"
+#include "Utility.h"
 
 //
 // Declarations

